tests/foreman/api/test_activationkey.py

```diff
@@ -5,7 +5,11 @@
 from nailgun import client, entities
 from requests.exceptions import HTTPError
 from robottelo.config import settings
-from robottelo.datafactory import valid_data_list
+from robottelo.datafactory import (
+    invalid_names_list,
+    valid_data_list,
+    valid_names_list,
```

Comment:
@sthirugn as far as I understand `valid_names_list` method should be used for test_user module (due specific rules for user names)